nisystemlink/clients/assetmanagement/models/_query_assets.py

```diff
@@ -0,0 +1,35 @@
+from enum import Enum
+from typing import List, Optional
+
+from nisystemlink.clients.core._uplink._json_model import JsonModel
+
+
+class OrderBy(Enum):
+    """Field by which assets can be ordered/sorted. If OrderBy is not specified, no sorting will applied."""
+
+    LAST_UPDATED_TIMESTAMP = "LAST_UPDATED_TIMESTAMP"
+
+
+class QueryAssetRequest(JsonModel):
```

Comment:
Please include totalCount as well. More details can be found in assetservice swagger